interface/CH.json

```diff
@@ -1497,7 +1497,7 @@
     "/BrugervendteSystemer/{id}/OIOSAMLProfile/{value}": {
       "put": {
         "summary": "Set the OIOSAMLProfile for this BrugervendtSystem (user system)",
-        "description": "Set the `OIOSAMLProfile` for this `BrugervendtSystem` (user system)",
+        "description": "Set the `OIOSAMLProfile` for this `BrugervendtSystem` (user system). The updated process rules of this endpoint are:</br></br>- If the current profile is OIOSAML2 and it is updated to OIOSAML3:</br>1. Convert the current AssuranceLevel value from NSIS and set the converted value to the NSISAssuranceLevel attribute:{1, 2} => https://data.gov.dk/concept/core/nsis/loa/Low, 3 => https://data.gov.dk/concept/core/nsis/loa/Substantial, and 4 => https://data.gov.dk/concept/core/nsis/loa/High.</br>2. Reset the AssuranceLevel to 0.</br></br>- If the current profile is OIOSAML3 and it is updated to OIOSAML2:</br>1. Convert the current NSISAssuranceLevel value to NIST and set the converted value to the AssuranceLevel attribute:https://data.gov.dk/concept/core/nsis/loa/Low => 2, https://data.gov.dk/concept/core/nsis/loa/Substantial => 3, and https://data.gov.dk/concept/core/nsis/loa/High => 4.</br>2. Reset the NSISAssuranceLevel to empty.",
```

Comment:
I would recommend a small update:
"description": "Set the `OIOSAMLProfile` for this `BrugervendtSystem` (user system). The processing rules are:</br></br>- If the current profile is OIOSAML2 and it is updated to OIOSAML3:</br>1. Convert the current AssuranceLevel value from NIST value to NSIS and set to the NSISAssuranceLevel attribute: {1, 2} => https://data.gov.dk/concept/core/nsis/loa/Low, 3 => https://data.gov.dk/concept/core/nsis/loa/Substantial, and 4 => https://data.gov.dk/concept/core/nsis/loa/High.</br>2. Reset the AssuranceLevel to 0.</br></br>- If the current profile is OIOSAML3 and it is updated to OIOSAML2:</br>1. Convert the current NSISAssuranceLevel value from NSIS to NIST and set the converted value to the AssuranceLevel attribute: https://data.gov.dk/concept/core/nsis/loa/Low => 2, https://data.gov.dk/concept/core/nsis/loa/Substantial => 3, and https://data.gov.dk/concept/core/nsis/loa/High => 4.</br>2. Reset the NSISAssuranceLevel to empty."

Reply:
Updated